lib/kitchen/directions/bake_injected_exercise/bake_injected_exercise.rb

```diff
@@ -1,17 +1,71 @@
 # frozen_string_literal: true
 
 module Kitchen::Directions::BakeInjectedExercise
-  def self.v1(exercise:)
-    V1.new.bake(exercise: exercise)
+  def self.v1(exercise:, alphabetical_multiparts: false)
+    V1.new.bake(exercise: exercise, alphabetical_multiparts: alphabetical_multiparts)
   end
 
   class V1
-    def bake(exercise:)
+    def bake(exercise:, alphabetical_multiparts:)
       question_count = exercise.injected_questions.count
       exercise[:'data-question-count'] = question_count
       exercise[:'data-is-multipart'] = question_count > 1 ? 'True' : 'False'
 
       context = exercise&.exercise_context
+      stimulus = exercise&.first("div[data-type='exercise-stimulus']")
+
+      # To handle alphabetical multipart questions without interrupting numbering
+      if alphabetical_multiparts && stimulus
+        solutions_clipboard = Kitchen::Clipboard.new
+        questions_clipboard = Kitchen::Clipboard.new
+        alphabet = *('a'..'z')
+        wrapper_id = "#{exercise.injected_questions.first.id}-wrapper"
```

Comment:
To link up with solution number, steals the first question id and appends '-wrapper' onto it